packages/config/config/devices/0x0131/ne-nas-pd07u1.json

```
@@ -0,0 +1,161 @@
+{
+	"manufacturer": "Zipato",
```

While this is called `manufacturer`, we're actually using it for the brand name.
```suggestion
	"manufacturer": "Alloy",
```

---

Ok, noted.  Wasn't sure if you wanted to align with the 0x0131 description